cf_xarray/tests/test_accessor.py

```diff
@@ -1091,3 +1091,43 @@ def test_stack(obj):
 
     actual = obj.cf.stack({"latlon": ["latitude", "longitude"]})
     assert_identical(expected, actual)
+
+
+def test_differentiate_follow_positive():
+    da = xr.DataArray(
+        np.arange(10)[::-1],  # like ocean temperature
+        dims="z",
+        coords={"z": ("z", np.arange(10), {"positive": "down"})},
+    )
+
+    expected = da.differentiate("z", 2)
+    actual = da.cf.differentiate("z", 2)
+    assert_identical(expected, actual)
+
+    expected = -1 * da.differentiate("z", 2)
+    actual = da.cf.differentiate("z", 2, follow_positive=True)
+    assert_identical(expected, actual)
+
+    with da.isel(z=slice(None, None, -1)) as da:
+        expected = -1 * da.differentiate("z", 2)
+        actual = da.cf.differentiate("z", 2, follow_positive=True)
+        assert_identical(expected, actual)
+
+    with xr.set_options(keep_attrs=True):
+        da["z"] = da.z * -1
```

Comment:
I think this particular test is right but please check!

Reply:
I think it's correct.